Add examples to server timing pages (#22605)
mdn/content: files/en-us/web/api/performanceservertiming/description/index.md

A string.

## Examples

### Logging server timing entries

> Given a {{HTTPHeader("Server-Timing")}} header with a description `"Cache Read"`:

Reviewer: ```suggestion
Use a {{HTTPHeader("Server-Timing")}} header with a description `"Cache Read"`.
```

Reviewer: I don't recommend putting an example in the middle of a sentence, no matter how short it is.

```http
Server-Timing: cache;desc="Cache Read";dur=23.2
```

> You can use a {{domxref("PerformanceObserver")}} to watch for {{domxref("PerformanceServerTiming")}} entries as they are recorded.

Reviewer: ```suggestion
Then use a {{domxref("PerformanceObserver")}} to watch for {{domxref("PerformanceServerTiming")}} entries as they are recorded.
```
